src/globalvars.ts

```diff
@@ -1,22 +1,15 @@
-import { inebrietyLimit, myAdventures, myInebriety, myTurncount } from "kolmafia";
-import { $item, have } from "libram";
-
 export const log = {
   initialEmbezzlersFought: 0,
   digitizedEmbezzlersFought: 0,
 };
 
-export const globalOptions: {
-  ascending: boolean;
-  stopTurncount: number | null;
-} = {
-  stopTurncount: null,
-  ascending: false,
-};
+export const globalOptions: { ascending: boolean; stopTurncount: number | null; noBarf: boolean } =
+  {
+    stopTurncount: null,
+    ascending: false,
+    noBarf: false,
+  };
 
-export function estimatedTurns(): number {
-  return globalOptions.stopTurncount
-    ? globalOptions.stopTurncount - myTurncount()
-    : (myAdventures() + (globalOptions.ascending && myInebriety() <= inebrietyLimit() ? 60 : 0)) *
-        (have($item`mafia thumb ring`) ? 1.04 : 1);
+export function clamp(n: number, min: number, max: number): number {
```

Comment:
Libram should export its utility functions now, so we can get rid of this export. That can be a job for another day though.